src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,105 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int ILLEGAL_AGE = 12;
+    private static final int LEGAL_AGE = 22;
+    private static final String ILLEGAL_PASSWORD_LENGTH = "Pass";
+    private static final String VALID_PASSWORD = "Password";
+    private static final String LOGIN = "Login";
+    private static RegistrationService registrationService;
+    private static StorageDao storageDao;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        Storage.people.clear();
+        user = new User();
+        user.setLogin(LOGIN);
+        user.setPassword(VALID_PASSWORD);
+        user.setAge(LEGAL_AGE);
+    }
+
+    @Test
+    void register_nullUser_notOk() {
+        user = null;
+        Throwable thrown = assertThrows(RuntimeException.class, () -> {
+            registrationService.register(user); });
+        assertEquals("User can’t be null", thrown.getMessage());
+    }
```

Comment:
we can remove redundant brackets 
`() -> registrationService.register(user)`
also, it is cool that you check the message 👍